linker/shared/src/main/scala/org/scalajs/linker/checker/IRChecker.scala

```diff
@@ -191,6 +191,17 @@ private final class IRChecker(unit: LinkingUnit, logger: Logger) {
           i"Duplicate definition of ${namespace.prefixString}method " +
           i"'$name' in class '${classDef.className}'")
     }
+
+    // JS native members
+    for {
+      (name, dupes) <- classDef.jsNativeMembers.groupBy(m => m.name.name)
```

Comment:
Consider dropping a comment that `jsNativeMembers` may only be in the public static namespace and that this is checked by assertion.

Reply:
Added :+1: